server/services/rekognition/rekognition.js

```diff
@@ -0,0 +1,26 @@
+const AWS = require('aws-sdk');
+var creds = new AWS.Credentials(process.env.AWS_ACCESS_KEY, process.env.AWS_SECRET_KEY); 
+var myConfig = new AWS.Config({
+    credentials: creds, region: "eu-west-1"
+});
+var rekognition = new AWS.Rekognition(myConfig);
+
+function assignLabels(imageData) {
+    AWS.region = "eu-west-1";
+    var params = {
+        Image: {
+            S3Object:{
+                Bucket: 'bristol-interns-project',
+                Name: "0aa220f92872282ef7cc2b1800527e99.jpg"
+            }
+        }
+    };
+    rekognition.detectLabels(params, function (err, data) {
+        if (err) {
+            console.log(err, err.stack);
+        } else {
+            const labels = data.Labels.map(label => label.Name)
+            console.log(labels);
```

Comment:
I'm not sure exactly what you're trying to log but you may find this dilutes the server logs a bit.